lib/rspec-puppet.rb

```diff
@@ -43,6 +43,7 @@ def self.current_example
   c.add_setting :default_node_params, :default => {}
   c.add_setting :default_trusted_facts, :default => {}
   c.add_setting :default_trusted_external_data, :default => {}
+  c.add_setting :facter_implementation, :default => :facter
```

Comment:
should probably change this to a string
```suggestion
  c.add_setting :facter_implementation, :default => 'facter'
```